### DISCUSSION TITLE
Welcome to XFM2-resources Discussions!

### DISCUSSION THREAD

**Discussion by xerhard** [xerhard/XFM2-resources#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Andy2No** [comment]
Is the source code available?  I haven't been able to find any, so far.

Do you have a replacement DAC in mind, now the Adafruit UDA1334A has been discontinued?
